### ISSUE TITLE
Support for manual sorting

### ISSUE THREAD

**Issue by vivek-dot-ca** [jillro/obsidian-cards-view-plugin#14]
Thank you for developing this wonderful plugin! My feature request would be to allow for manual sorting of cards in the card view. I am a long time Scrivener user and like to use its "cork board" feature to move cards around and organize a writing project. Having a similar ability to sort cards according to whatever order the user wants would make this an incredibly useful plugin for a wide range of folks! Thanks for considering this request!

**Comment by greetclammy** [comment]
Just in case you don't know, Obsidian has a core feature called *Canvas* that sound exactly like what you describe!

**Comment by tu2-atmanand** [comment]
Yes, the Canvas is a feature inside Obsidian to arrange your notes in cared format manually. Plugins are made to make your life easier by having a dynamic behavior or having some automation workflow, which this plugin has done a marvelous job interms of showing the recent updated notes-cards on top.

I understand, the UI and you might find, few other small features in this plugin might be better than the Canvas, thats why you want the static behavior in this plugin, but it will be again kind of like redeveloping the wheel.

Said that, your requested feature can still be implemented in this, by having a new option in the sorting dropdown input field as **manual**, so that user can manually sort the cards.
